docs/developer/architecture/core/logging-configuration-migration.asciidoc

```diff
@@ -0,0 +1,68 @@
+[[logging-configuration-migration]]
+== Logging configuration migration
+
+Compatibility with the legacy logging system is assured until the end of the `v7` version. 
+All log messages handled by `root` context are forwarded to the legacy logging service. If you re-write
+root appenders, make sure that it contains `default` appender to provide backward compatibility.
+
+NOTE: When you switch to the new logging configuration, you will start seeing duplicate log entries in both formats. 
+These will be removed when the `default` appender is no longer required. If you define an appender for a logger, 
+the log messages aren't handled by the `root` logger anymore and are not forwarded to the legacy logging service.
+
+[[logging-pattern-format-old-and-new-example]]
+[options="header"]
+|===
+
+| Parameter | Platform log record in **pattern** format | Legacy Platform log record **text** format
+
+| @timestamp | ISO8601_TZ `2012-01-31T23:33:22.011-05:00` | Absolute `23:33:22.011`
+
+| logger | `parent.child` | `['parent', 'child']`
+
+| level | `DEBUG` | `['debug']`
+
+| meta | stringified JSON object `{"to": "v8"}`| N/A
+
+| pid | can be configured as `%pid` | N/A
+
+|===
+
+[[logging-json-format-old-and-new-example]]
+[options="header"]
+|===
+
+| Parameter | Platform log record in **json** format | Legacy Platform log record **json** format
+
+| @timestamp | ISO8601_TZ `2012-01-31T23:33:22.011-05:00` | ISO8601 `2012-01-31T23:33:22.011Z`
+
+| logger | `log.logger: parent.child` | `tags: ['parent', 'child']`
+
+| level | `log.level: DEBUG` | `tags: ['debug']`
+
+| meta | separate property `"meta": {"to": "v8"}` | merged in log record  `{... "to": "v8"}`
+
+| pid | `process.pid: 12345` | `pid: 12345`
+
+| type | N/A | `type: log`
+
+| error | `{ message, name, stack }` | `{ message, name, stack, code, signal }`
+
+|===
+
+[[logging-cli-migration]]
+=== Logging configuration via CLI migration guide
+
+NOTE: The `--verbose` and `--silent` cli flags are supported and have build-in behavior. You can override the default behavior using the following equivalent platform logging configurations:
+[options="header"]
+|===
+
+| legacy logging | {kib} Platform logging | cli shortcuts
+
+|--verbose| --logging.root.level=debug --logging.root.appenders[0]=default --logging.root.appenders[1]=console | --verbose
```

Comment:
Optional: Let's remove `--logging.root.level=debug --logging.root.appenders[0]=default --logging.root.appenders[1]=console` or present it in `yml` format. Otherwise, it's not clear whether we recommend using these cli options.

Reply:
That makes sense.

Reply:
@mshustov using the platform logging flags as cli args for verbose logs has slightly different behavior to simply using the `--verbose` shortcut. We also don't support the `--quiet` flag anymore and it would be nice to give users and alternative.

This table is also specifically aimed at logging configuration via CLI, hence the inline config.

Reply:
ok, makes sense